source/agora/common/Config.d

```diff
@@ -281,6 +281,12 @@ public struct AdminConfig
 
     /// Bind port
     public ushort port = 0xB0B;
+
+    /// Username
+    public string username = "admin";
+
+    /// Password
+    public string password = "s3cr3t";
```

Comment:
No default please

Reply:
Also it's not being parsed, so it always uses the default

Reply:
Removed